lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -4301,11 +4303,11 @@ hasReturnCollateral (Cardano.Tx (Cardano.TxBody content) _) =
 --
 -- NOTE: We don't have a 'Cardano.ProtocolParameters -> ProtocolParameters'
 -- function, so we need to manually ensure the hard-coded values are consistent.
-mockProtocolParametersForBalancing
-    :: (ProtocolParameters, Cardano.ProtocolParameters)
-mockProtocolParametersForBalancing = (mockProtocolParameters, nodePParams)
-  where
-    nodePParams = Cardano.ProtocolParameters
+mockProtocolParametersForBalancing ::
+    (ProtocolParameters, Cardano.ProtocolParameters)
```

Comment:
No need to reformat the type signature here, as it was actually already in our standard format, which is:

```suggestion
mockProtocolParametersForBalancing
    :: (ProtocolParameters, Cardano.ProtocolParameters)
```

(with `::` on a newline)
(See https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md#avoid-variable-length-indentation)

Reply:
One unfortunate side effect is that syntax highlighting doesn't recognize function name as such and doesn't highlight it when `::` is on the next line.